interface/modules/custom_modules/oe-module-claimrev-connect/public/setup.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/**
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+    require_once "../../../../globals.php";
+
+    $ta = "setup";
+    use OpenEMR\Modules\ClaimRevConnector\ClaimRevModuleSetup;
```

Comment:
Change above to following to support the acl check:
```
require_once "../../../../globals.php";

use OpenEMR\Common\Acl\AclMain;
use OpenEMR\Common\Twig\TwigContainer;
use OpenEMR\Modules\ClaimRevConnector\ClaimRevModuleSetup;

$tab = "setup";

//ensure user has proper access
if (!AclMain::aclCheckCore('admin', 'manage_modules')) {
    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl("ClaimRev Connect - Setup")]);
    exit;
}
```

Reply:
added